sonar-batch-protocol/src/main/java/org/sonar/batch/protocol/ProtobufUtil.java

```diff
@@ -44,6 +39,43 @@ private ProtobufUtil() {
     }
   }
 
+  public static <MESSAGE extends Message> Iterable<MESSAGE> readFileMessages(File file, final Parser<MESSAGE> parser) {
```

Comment:
Personally I don't mind but some people prefer that generic parameter is named with a single letter. So maybe consider renaming MESSAGE -> M